src/Carousel.vue

```diff
@@ -303,7 +303,7 @@ export default {
       return (this.offset + this.dragOffset) * -1;
     },
     isHidden() {
-      return this.carouselWidth <= 0;
+      return this.carouselWidth;
```

Comment:
It actually looks as if we're not using this at all, so we can remove it if you'd prefer. If you want to keep it, we'll need to make a comparison here ( `this.carouselWidth` equals or doesn't equal something ) as it has to return a `boolean` value.

Reply:
Yea i just left this part of code for safety reasons. Seemed to me that it is not used but if it were then  would be good to still return the caruselWidth so that could make the element visible as hiding of elements was problem here.  But as i couldn't see  this part of code working so didn't know in which way it is used so i just left it there.  And as you say it has to return boolean value then would be better to remove it from the code especially as it is not used at all.